tests/openshell/kata/test_openshell_kata.py

```
@@ -0,0 +1,178 @@
+                            host="github.com",
+                            port=443,
+                            access="read-only",
+                            protocol="rest",
+                            enforcement="enforce",
+                        )
+                    ],
+                    binaries=[sandbox_pb2.NetworkBinary(path="/usr/bin/curl")],
+                ),
+            ),
+        )
+        update_resp = sandbox_client._stub.UpdateConfig(  # noqa: FCN001
```

Ruff (0.16.1)

[warning] Invalid rule code in `# noqa`: FCN001

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)